src/components/sponsors/PastSponsors.tsx

```diff
@@ -1,10 +1,162 @@
-import React from "react";
+import amazonIcon from "@/assets/sponsors/amazon-text.png";
+import googleIcon from "@/assets/sponsors/google-text.png";
+import drracketIcon from "@/assets/sponsors/drracket-text.png";
+import microsoftIcon from "@/assets/sponsors/microsoft-text.png";
 
-const PastSponsors: React.FC = () => {
+type StaticImageData = {
+  src: string;
+};
+
+export const PastSponsors: React.FC = () => {
   return (
-    <>
-      <div className="text-lg">PastSponsors</div>
-    </>
+    <div className="flex flex-col md:w-3/4 mx-auto">
+      <h2 className="text-5xl md:text-7xl my-10 text-center text-primary">
+        Past Sponsors
+      </h2>
+      <div className="flex overflow-hidden">
+        <div className="flex space-x-2 ml-2 md:ml-8 md:space-x-8 animate-loop-scroll">
+          <a
+            href="https://www.amazon.com"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
```

Comment:
Nextjs recommends using the <Image /> component provided by nextjs because it automatically optimzes images - also that way the source can just be the image variable instead of image.src